epoch_broadcaster/src/server.rs

```diff
@@ -25,40 +33,175 @@ impl EpochBroadcaster for ProtoServer {
         &self,
         request: Request<SetEpochRequest>,
     ) -> Result<Response<SetEpochResponse>, TStatus> {
+        let reply = SetEpochResponse {};
         let current_epoch = self.server.epoch.load(SeqCst);
+        if current_epoch == 0 {
+            // Can't accept the RPC, since we don't know if it is stale.
+            // TODO(tamer): on startup, the broadcaster should sync with
+            // the epoch service and read the latest epoch.
+            return Ok(Response::new(reply));
```

Comment:
Shouldn't this be an error?

Reply:
It is safer to make this an error indeed, will do.